src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -22,30 +22,114 @@
 *
 * Author: Ken Zangelin
 */
-
+#include <sys/time.h>
```

Comment:
Or maybe I'm wrong... this is probably used at semTake() to measure time waited at sem.